docs/usage/configuration-options.md

```diff
@@ -1028,6 +1028,18 @@ If you wish for Renovate to process only select paths in the repository, use `in
 Alternatively, if you need to just _exclude_ certain paths in the repository then consider `ignorePaths` instead.
 If you are more interested in including only certain package managers (e.g. `npm`), then consider `enabledManagers` instead.
 
+## internalChecksFilter
+
+This setting determines whether Renovate controls when and how filtering of internal checks are performed, particularly when multiple versions of the same update type are available.
+Currently this applies to the `stabilityDays` check only.
+
+- `none`: No filtering will be performed, and the highest release will be used regardless of whether it's pending or not
+- `strict`: All pending releases will be filtered if they are pending. PRs will be skipped unless a non-pending version is available
```

Comment:
```suggestion
- `strict`: All pending releases will be filtered. PRs will be skipped unless a non-pending version is available
```

I think "all pending releases will be filtered" doesn't need the "if they are pending" qualifier?